### ISSUE TITLE
[Small Feature Request] Add `class` attribute to the `@qualifier` command in the HTML output

### ISSUE THREAD

**Issue by parmi93** [doxygen/doxygen#11245]
I am trying to achieve something like this:

![image](https://github.com/user-attachments/assets/890d940e-a664-44fb-867f-a927e907a872)

where the `@qualifier Thread-safe` has a different `background-color` than `inline`.
Unfortunately this is not possible because all qualifiers are rendered as `<span class="mlabel">...</span>`

I would like to purpose to change this command into something like that:

`\qualifier <name> | "(text)" [<label>]` --> `<span class="mlabel label">name | "(text)"</span>`

where `<name> | "(text)"` is displayed to the user, and `<lablel>` is used as the value for the `class` attribute.

**Comment by albert-github** [comment]
The current syntax is
```
\qualifier <label> | "(text)"
```
the proposed syntax is nice but I'm thinking more in the direction of:
```
\qualifier['{'option[,option]∗'}'] <label> | "(text)"
```
where `option` can be a class label.

**Comment by albert-github** [comment]
Nearly, this is indeed how you would use it but without the signal quotes (the `'{'` sifbals that the '{' is mandatory when using options:
```
\qualifier Thread-safe --> <span class="mlabel">Thread-safe</span>
\qualifier{my_class1,my_class2} Thread-safe --> <span class="mlabel my_class1 my_class2">Thread-safe</span>
```

**Comment by parmi93** [comment]
Thanks for the clarification, I think this new feature might be fine =)

**Comment by parmi93** [comment]
Maybe I'm asking a bit too much now =)
Would it be possible to add support for the `@ref` tag when used inside the `<lablel> | "text"` of the `@qualifier` command?

What I'm trying to achieve is to create a clickable qualifier, that takes the user to the HTML page where the meaning of Thread-safe is explained, I tried to use the `'@qualifer` command like this, but it didn't work =(

`@qualifier "@ref thread_safe \"Thread-safe\""`

**Comment by albert-github** [comment]
`\ref` Interesting thought but a bit out of scope for this issue, just create another issue for it.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #11250

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
